.github/workflows/publish.yml

```
@@ -0,0 +1,17 @@
+on: [push]
+
+permissions:
+  contents: write
+jobs:
+  publish-guidelines:
+    runs-on: ubuntu-latest
+    name: Publish Logseq Graph
+    steps:
+      - uses: actions/checkout@v4
+      - uses: logseq/publish-spa@v0.3.0
+      - name: Add a nojekyll file # to make sure asset paths are correctly identified
+        run: touch $GITHUB_WORKSPACE/www/.nojekyll
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@v4
+        with:
+          folder: www
```

We don't need to record the history of the rendered site.

```suggestion
          folder: www
          single-commit: true
```